staging/src/k8s.io/metrics/IMPLEMENTATIONS.md

```diff
@@ -2,10 +2,6 @@
 
 ## Resource Metrics API
 
-- [Heapster](https://github.com/kubernetes/heapster): a application which
-  gathers metrics, writes them to metrics storage "sinks", and exposes the
-  resource metrics API from in-memory storage.
```

Comment:
Can you mark this as deprecated rather than removing this?

Also, while you're in this file, can you update the link for Prometheus Adapter to https://github.com/kubernetes-sigs/prometheus-adapter ?

Reply:
Thanks @ehashman , done.